vspace/src/main/java/edu/asu/diging/vspace/web/staff/ModuleController.java

```diff
@@ -16,10 +17,11 @@
 
     @RequestMapping("/staff/module/{id}")
     public String showModule(@PathVariable String id, Model model) {
-
+        ModuleStatus status = moduleManager.getModule(id).getModuleStatus();
         model.addAttribute("module", moduleManager.getModule(id));
         model.addAttribute("slides", moduleManager.getModuleSlides(id));
         model.addAttribute("sequences", moduleManager.getModuleSequences(id));
+        model.addAttribute("moduleStatus", status);
```

Comment:
get the module once and then use the module in line 21 and 24.

Reply:
Done